src/components/home/components/styles.css

```diff
@@ -85,6 +85,11 @@
     bottom: 30px;
     background-color: #f56a00;
 }
+@media only screen and (max-width: 700px) {
+    .ant-menu-dark{
+        margin-left: -15%;
```

Comment:
this might be over kill and might have some un-intended consequences in the future. You similar has to adjust the `margin-left` of the logo its self

Reply:
Given that it's applied to only screen size of 700 and below, I think it's side effects can easily be figured out